java/src/org/openqa/selenium/docker/v1_41/CreateContainer.java

```diff
@@ -52,6 +52,7 @@ public CreateContainer(DockerProtocol protocol, HttpHandler client) {
   }
 
   public Container apply(ContainerConfig info) {
+    this.protocol.getImage(info.getImage().getName());
```

Comment:
But this is doing a `docker pull`, isn't it? 

If so, it is not ideal because it implies an extra network call that can slow down the process. Can we do a `docker inspect` instead? https://stackoverflow.com/a/33061675

Reply:
I see it doesn't jump to API to pull immediately. Based on image name, it does this first
`new HttpRequest(GET, String.format("/v%s/images/json", DOCKER_API_VERSION))`
So, I guess this is the inspect that you are looking for?

Reply:
The implementation is in these lines https://github.com/SeleniumHQ/selenium/blob/ec5757d2e9dfe12dfdfe73d9ade8e509301ccd18/java/src/org/openqa/selenium/docker/v1_41/V141Docker.java#L67-L90

Reply:
Oh, it was already implemented. Great!